### PR TITLE
Enhance language dropdown ordering

### PR DESCRIPTION
**Pull request by dipesh-rawat**
This PR enhances the arrangement of localized language choices within the navbar dropdown menu. The improvement is achieved through the implementation of alphabetical sorting based on English equivalents, by utilizing the `weight` configuration value in `hugo.toml` for the respective languages. The changes contribute to a more organized and user-friendly navigation experience.

[Preview Page](https://deploy-preview-44869--kubernetes-io-main-staging.netlify.app/)

### CODE REVIEW

**Comment by sftim** [comment]
Thanks
Should we document this detail in https://kubernetes.io/docs/contribute/localization/ as well?

File: content/en/docs/contribute/localization.md [hunk 1]
```diff
@@ -187,8 +187,11 @@ script and use it in the theme. Assign "language name in latin script" to
 `languageNameLatinScript`. For example, `languageNameLatinScript ="Korean"` or
 `languageNameLatinScript = "Deutsch"`.
 
-When assigning a `weight` parameter for your block, find the language block with
-the highest weight and add 1 to that value.
+The `weight` parameter determines the order of languages in the language selection bar.
+A lower weight takes precedence, resulting in the language appearing first. 
+When assigning the `weight` parameter, it is important to examine the existing languages 
+block and adjust their weights to ensure they are in a sorted order relative to all languages,
```
Comment (T-Lakshmi) [inline content/en/docs/contribute/localization.md:193]: Why can't we reserve some buffer numbers in between that may be useful in the future for upcoming languages?
This way we can avoid existing code edit.

I hope my question is valid.

Comment (dipesh-rawat) [reply content/en/docs/contribute/localization.md:193]: It's a valid consideration, one that I even pondered while implementing this change. But I feel maintaining a sorted order list _(proper sequenced weight)_ for languages does seem more practical in the long run. Predicting the exact number of future languages can be challenging, and allocating buffer numbers could lead to uneven distribution over time, potentially causing confusion. One more thing I will do _(now)_ is organize language blocks itself in sorted order, which would help in keeping things organized and easier to manage in the long run.

**Comment by sftim** [comment]
Thanks

/approve

File: content/en/docs/contribute/localization.md (hunk 1 above)
Comment (T-Lakshmi) [reply content/en/docs/contribute/localization.md:193]: Agree with this,
Thanks.

**Pull request merged**